app/src/main/kotlin/com/wire/android/feature/auth/registration/personal/email/CreatePersonalAccountEmailViewModel.kt

```diff
@@ -3,21 +3,35 @@ package com.wire.android.feature.auth.registration.personal.email
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.wire.android.core.exception.Failure
+import com.wire.android.core.usecase.DefaultUseCaseExecutor
+import com.wire.android.core.usecase.UseCaseExecutor
+import com.wire.android.shared.user.email.ValidateEmailError
+import com.wire.android.shared.user.email.ValidateEmailParams
+import com.wire.android.shared.user.email.ValidateEmailUseCase
+import kotlinx.coroutines.Dispatchers
 
 //TODO: add use cases & layers below
-class CreatePersonalAccountEmailViewModel : ViewModel() {
+class CreatePersonalAccountEmailViewModel(
```

Comment:
I think we're missing some test cases for this class

Reply:
good catch again, done.